skills.pl

```diff
@@ -33,20 +33,20 @@
     write('You have chosen class: '), write(ChosenClass), nl.
     
 try_choose_class(_) :-
-	write('Incorrect choice, try other option'), nl,
+	println('Incorrect choice, try other option'),
 	choose_your_class.
     
 print_class_information(N) :-
 	write('Class: '), write(N), nl,
 	skills(hero_class(N), S, D, A),
-	write(S), nl,
-	write(D), nl,
-	write(A), nl,
+	println(S),
+	println(D),
+	println(A),
 	write('Type: "'), write(N), write('." to choose this class'), nl.
 	
-levelUp :- 
-	write('You have increased your level.'), nl,
-	printCurrentStats,
+level_up :- 
+	println('You have increased your level.'),
+	print_current_stats,
 	findall(Stat, stat_bonus(Stat, _), Stats),
```

Comment:
I believe findall + maplist is equivalent to foreach, which is shorter and can be used here and in other places.